roles/common/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+postgres_initdb_args: '--auth-host=scram-sha-256'
+postgres_host_auth_method: 'scram-sha-256'
+ldap_cacert_ca_crt: ''
+bundle_ca_crt: ''
+projects_existing_claim: ''
+supported_pg_version: 13
```

Comment:
Is this needed since we don't use any of those variables in the common role ?
If we really need them then we should also remove them from the installer role.

Reply:
Nope, I will remove.  Thanks!